content/cohorts/_index.en.adoc

```diff
@@ -2,6 +2,6 @@
 title: "Cohorts"
 icon: "vr-cardboard"
 description: "Explore UNICEF Venture Fund start-up companies and Open Source projects by cohort."
-type: "team"
+type: "docs"
```

Comment:
I haven't run these changes locally yet, but I am curious what will happen to the _teams_ layout we are using presently?

![Screenshot of the _teams_ layout used for the Cohorts page in the UNICEF Open Source Inventory site.](https://user-images.githubusercontent.com/4721034/178276898-81e32911-7b8f-4a56-8423-60987c35c762.png "Screenshot of the _teams_ layout used for the Cohorts page in the UNICEF Open Source Inventory site.")

Reply:
Reverted in commit 6c4ad0bc6c1863c8ba8e10fe41947d10641ec6e8.